### ISSUE TITLE
Getting an error with baseURL dereferencing from .bin/jspm-dev-server

### ISSUE THREAD

**Issue by kwesterfeld** [mikz/jspm-dev-server#9]
I am able to use version 0.3.1 of jspm-dev-server with the latest JSPM beta, but the 0.4.0 version fails with:

```
> jspm-dev-server --proxy http://localhost:8182 --port 8183 --ignore smc/jspm_packages

...ui/node_modules/jspm-dev-server/bin/jspm-dev-server:31
options.baseURL = jspmConfig.loaderBrowser.baseURL
                                          ^

TypeError: Cannot read property 'baseURL' of undefined
    at Object.<anonymous> (...ui/node_modules/jspm-dev-server/bin/jspm-dev-server:31:43)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)
    at startup (node.js:139:18)
    at node.js:999:3
```

**Comment by mikz** [comment]
That was reported and fixed as #8. Released as 0.4.1.

**Comment by kwesterfeld** [comment]
Thanks!
